src/Bicep.Cli/Commands/PublishCommand.cs

```diff
@@ -45,6 +50,18 @@ public async Task<int> RunAsync(PublishArguments args)
             var inputUri = PathHelper.FilePathToFileUrl(inputPath);
             var configuration = this.configurationManager.GetConfiguration(inputUri);
             var moduleReference = ValidateReference(args.TargetModuleReference, configuration);
+
+            if (PathHelper.HasExtension(inputUri, LanguageConstants.JsonFileExtension) ||
+                PathHelper.HasExtension(inputUri, LanguageConstants.JsoncFileExtension) ||
+                PathHelper.HasExtension(inputUri, LanguageConstants.ArmTemplateFileExtension))
```

Comment:
Do we have the condition with the 3 extensions OR'd together somewhere else. Might be worth refactoring if we do.

Reply:
Yeah good point. I'll create a helper method in `PathHelper`.